stellargraph/mapper/sequences.py

```diff
@@ -527,3 +530,128 @@ def __len__(self):
 
     def __getitem__(self, index):
         return self.inputs, self.targets
+
+
+@experimental(reason="Missing unit tests.")
```

Comment:
A user doesn't construct these directly, so to avoid getting extra warnings (i.e. a warning from both `GraphGenerator(...)` and `gen.flow(...)`), it may not be necessary to annotate this one:

```suggestion
```

Reply:
Removed as suggested although in theory anyone could implement a custom generator that returns a `GraphSequence`. I would guess that the likelihood of that happening before we have unit tests is rather low so best to avoid the clutter of all the warnings as suggested.

Reply:
Yeah, it's unlikely, and, we don't seem to encourage or support users using our `Sequence`s, e.g. they're not in our documentation at all.